src/grpc.ts

```diff
@@ -169,8 +169,16 @@ export class GrpcClient {
       this.grpc = options.grpc!;
       this.grpcVersion = '';
     } else {
-      this.grpc = require('grpc');
-      this.grpcVersion = require('grpc/package.json').version;
+      // EXPERIMENTAL: If GCLOUD_USE_GRPC_JS is set, use the JS-based
+      // implementation of the gRPC client instead. Requires http2 (Node 8+).
+      if (semver.satisfies(process.version, '8.x') &&
```

Comment:
I think a while back, the env vars shifted from "GCLOUD" to "GOOGLE_CLOUD" to let the "gcloud" SDK sustain its own identity. I don't have any strong opinions on it, just mentioning for consideration.